testkit/src/main/scala/org/bitcoins/testkit/fixtures/DLCOracleDAOFixture.scala

```diff
@@ -0,0 +1,43 @@
+package org.bitcoins.testkit.fixtures
+
+import org.bitcoins.crypto.AesPassword
+import org.bitcoins.dlc.oracle.DLCOracleAppConfig
+import org.bitcoins.dlc.oracle.storage._
+import org.bitcoins.testkit.EmbeddedPg
+import org.bitcoins.testkit.keymanager.KeyManagerTestUtil.bip39PasswordOpt
+import org.scalatest._
+
+import scala.concurrent.Future
+
+case class DLCOracleDAOs(
+    rValueDAO: RValueDAO,
+    eventDAO: EventDAO,
+    outcomeDAO: EventOutcomeDAO)
+
+trait DLCOracleDAOFixture extends BitcoinSFixture with EmbeddedPg {
+
+  implicit protected def config: DLCOracleAppConfig
```

Comment:
Is there a reason you don't just move the implementation up here? It seems like you are use the same implementation in all places? 

```scala
BitcoinSTestAppConfig.getDLCOracleWithEmbeddedDbTestConfig(pgUrl)
```